### ISSUE TITLE
beaupy.prompt() crashes when typing "\".

### ISSUE THREAD

**Issue by therealOri** [petereon/beaupy#104]
# About:
When typing `\` in the terminal/powershell, etc. when the secure prompt flag is off/false, It will just straight up crash. I believe `rich` is the culprit but I am not knowledgeable enough with it yet to know if there is a fix or a workaround to it.
__ __

<br>

# Traceback
```bash
Traceback (most recent call last):
  File "/home/ori/Documents/GitHub/Poly_locker/polyLock.py", line 463, in <module>
    main()
  File "/home/ori/Documents/GitHub/Poly_locker/polyLock.py", line 168, in main
    file_path = beaupy.prompt("Path to the file you want to obfuscate/lock.")
                ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/ori/Documents/GitHub/Poly_locker/cmiENV/lib/python3.12/site-packages/beaupy/_beaupy.py", line 249, in prompt
    element.state = new_state
    ^^^^^^^^^^^^^
  File "/home/ori/Documents/GitHub/Poly_locker/cmiENV/lib/python3.12/site-packages/questo/abstract/abstract_element.py", line 76, in state
    self.update()
  File "/home/ori/Documents/GitHub/Poly_locker/cmiENV/lib/python3.12/site-packages/questo/abstract/abstract_element.py", line 81, in update
    self._live.update(renderable=rendered)
  File "/home/ori/Documents/GitHub/Poly_locker/cmiENV/lib/python3.12/site-packages/rich/live.py", line 214, in update
    renderable = self.console.render_str(renderable)
                 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/ori/Documents/GitHub/Poly_locker/cmiENV/lib/python3.12/site-packages/rich/console.py", line 1430, in render_str
    rich_text = render_markup(
                ^^^^^^^^^^^^^^
  File "/home/ori/Documents/GitHub/Poly_locker/cmiENV/lib/python3.12/site-packages/rich/markup.py", line 167, in render
    raise MarkupError(
rich.errors.MarkupError: closing tag '[/black on white]' at position 65 doesn't match any open tag
```
__ __

**Comment by petereon** [comment]
Hey @therealOri, long time no see. 

I think you are correct, `\` character acts as an escape-character for a square brackets that begins the definition of a `rich` style. I'll think about a fix. Should release today.

**Comment by therealOri** [comment]
Long time indeed!

Also, sweet!

**Comment by petereon** [comment]
Released as [3.9.1](https://pypi.org/project/beaupy/3.9.1/)
